src/lsp/server/utils/objectInfoUtils.ts

```diff
@@ -75,11 +75,11 @@ class OrgInfo {
 }
 
 /**
- * Utility class for Org
+ * Utility class for managing ObjectInfo
  * - fetching data and cache it in L1 and L2 for authorized org
  * - wipe out data when logout.
  */
-export class OrgUtils {
+export class ObjectInfoUtils {
```

Comment:
Renamed the class to reflect what it manages instead of where it came from.

Reply:
Renaming the class makes sense but not the file. The file contains more than one class, which perform org-related tasks as a whole. So I think that we should have the file as OrgUtils still.

Or break out the 2 classes into their own files, especially since `ObjectInfoUtils` class is rather large?

Reply:
There is no need to rename the files or class names. OrgUtils encompasses more functionality than just fetching ObjectInfo. The ObjectInfo functions can be viewed as a subset of OrgUtils' capabilities.

Reply:
Just another reminder that there is a class call OrgUtils in utils folder of the root project too that is absolutely unrelated. I will revert the names. It's strange, to me, that there are two separate OrgUtils.